tests/Bundle/ContaoCoreBundleTest.php

```diff
@@ -81,6 +81,7 @@ public function testBuild()
                 'Contao\\CoreBundle\\DependencyInjection\\Compiler\\AddPackagesPass',
                 'Contao\\CoreBundle\\DependencyInjection\\Compiler\\AddResourcesPathsPass',
                 'Contao\\CoreBundle\\DependencyInjection\\Compiler\\SetApplicationPass',
+                'Contao\\CoreBundle\\DependencyInjection\\Compiler\\CreateFrameworkInitializerEventDispatcherPass'
```

Comment:
This looks like it could be removed together with the `CreateFrameworkInitializerEventDispatcherPassTest` class?